### DISCUSSION TITLE
When I use nest create on one-to-many relation, inconsistent creation order

### DISCUSSION THREAD

**Discussion by myNameIsDu** [prisma/prisma#14351] [Q&A]
<img width="1299" alt="image" src="https://user-images.githubusercontent.com/53305259/179456973-ce6571f3-edd6-494e-a5e8-59341059ab2d.png">

<img width="488" alt="image" src="https://user-images.githubusercontent.com/53305259/179457069-6fff5b44-65c0-4075-870a-e9ec68e7c2d8.png">
That's strange, The order of the data in the outer layer is regular, And I know I shouldn’t depend on this, So I add a `sort` field, But I'm curious why that is.

**Comment by well-balanced** [comment]
could you show me your full code lines?

**Reply by myNameIsDu** [reply]
Thanks for helping me. Here is my full code.

<img width="715" alt="image" src="https://user-images.githubusercontent.com/53305259/179880081-a82ba894-9e31-4aa9-ad5c-fb9c449af75c.png">

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
